hasjob/templates/reject_domain_email.html

```diff
@@ -0,0 +1,15 @@
+{% extends "inc/email_layout_lite.html" %}
+
+{% block content %}
+  <p>Hi {{ post.company_name }},</p>
+  <p>This is regarding your job post on Hasjob titled "<strong>{{ post.headline }}</strong>". Unfortunately, your post is in violation of our <a href="{{ url_for('terms_of_service', _external=true) }}">Terms of Service</a> and has been <strong>banned</strong>. As a result all the job posts under the same domain has been taken down and your user account has been banned too. Your post was reviewed by {{ post.reviewer.fullname }}, who left the following comment:</p>
+  <p>"<strong>{{ post.review_comments }}</strong>"</p>
+  <p>Here is a list of all the jobs posts that have been taken down:</p>
+  <ul>
+    {% for jobpost in post.domain.jobposts %}
```

Comment:
This will include drafts and previously rejected posts. This list should be passed in to the template.